pkg/blockdevice/v1alpha2/blockdevice.go

```diff
@@ -0,0 +1,178 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	ndmapisv1alpha1 "github.com/openebs/maya/pkg/apis/openebs.io/ndm/v1alpha1"
+	apisv1alpha1 "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+)
+
+//TODO: While using these packages UnitTest must be written to corresponding function
+
+// BlockDeviceState is label for block device states
+type BlockDeviceState string
+
+const (
+	// BlockDeviceStateActive is active state of the block device
+	BlockDeviceStateActive BlockDeviceState = "Active"
+)
+
+// DefaultBlockDeviceCount is a map containing the default block device count of various raid types.
+var DefaultBlockDeviceCount = map[string]int{
```

Comment:
This is duplicated in multiple packages

Reply:
:+1:

Reply:
Will remove from `/pkg/algorithm/nodeselect/v1alpha1` in PR: #1227